Changelog.md

```diff
@@ -43,6 +44,7 @@ Bugfixes:
 Build System:
  * Soltest: Add support for arrays in function signatures.
  * Soltest: Add support for struct arrays in function signatures.
+ * Soltest: Add support for left-aligned, unpadded hex string literals.
```

Comment:
Is this in 0.5.6 on purpose?

Reply:
Yes! That's the entry we merged into the 0.5.5 section by accident.